GVFS/GVFS.Common/Prefetch/BlobPrefetcher.cs

```diff
@@ -513,9 +479,59 @@ protected void DownloadMissingCommit(string commitSha, GitObjects gitObjects)
             }
         }
 
+        private static bool TryLoadFileOrFolderList(Enlistment enlistment, string valueString, string listFileName, bool isFolder, List<string> output, Func<string, string> elementValidationFunction, out string error)
+        {
+            output.AddRange(
+                valueString.Split(new[] { ';' }, StringSplitOptions.RemoveEmptyEntries)
+                .Select(path => BlobPrefetcher.ToAbsolutePath(enlistment, path, isFolder: isFolder)));
+
+            if (!string.IsNullOrWhiteSpace(listFileName))
+            {
+                IEnumerable<string> linesFromFile = null;
+                if (File.Exists(listFileName))

```

Comment:
I see here that if the `stdin` file exists, then we will never read from `stdin`, if you needed an extra reason to create `--stdin-files` instead.